### PR TITLE
Add Satosa role metadata ui elements configuration

### PR DESCRIPTION
**Pull request by mrvanes**

### CODE REVIEW

File: roles/satosa/defaults/main.yml [hunk 1]
```diff
@@ -13,4 +13,13 @@ satosa_port: "8080"
...
 satosa_user_id_hash_salt: "59o3gaxfwg0svem1n1g3tk22i9mihcom"
 
+satosa_mdui_display_name: SATOSA Display Name
+satosa_mdui_description: SATOSA Description
+satosa_mdui_information_url: "http://information.url/"
+satosa_mdui_privacy_statement_url: "http://privacy.url/"
+satosa_mdui_keywords: "[{lang: nl, text: [sleutelwoord1, sleutelwoord2]}, {lang: en, text: [keyword1, keyword2]}]"
+satosa_mdui_logo: "http://logo.url/"
+satosa_mdui_logo_width: "100"
+satosa_mdui_logo_height: "100"
```
Comment (baszoetekouw) [inline roles/satosa/defaults/main.yml]: I think display_name, description and keywords should be separately configurable for the frontend and backend

File: roles/satosa/templates/saml2_backend.yaml.j2 [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
 config:
   sp_config:
     key_file: certs/backend.key
     cert_file: certs/backend.crt
-    organization: {display_name: {{ environment_string }} Proxy SP, name: {{ environment_string }} Proxy SP, url: 'https://{{ hostnames.meta }}/'}
+    organization:
+      display_name: {{ environment_string }} Proxy SP
+      name: {{ environment_string }} Proxy SP
+      url: 'https://{{ hostnames.meta }}/'
     contact_person:
```
Comment (baszoetekouw) [inline roles/satosa/templates/saml2_backend.yaml.j2]: organization metadata should be fully configurable, too.  The values here "{{environment_strinf}} Proxy SP" etc can be the default values.

File: roles/satosa/templates/saml2_backend.yaml.j2 [hunk 3]
```diff
@@ -1,27 +1,45 @@
...
-    - {contact_type: technical, email_address: technical@{{ hostnames.proxy }}, given_name: Technical}
-    - {contact_type: support, email_address: support@{{ hostnames.proxy }}, given_name: Support}
+      - contact_type: technical
+        email_address: technical@{{ hostnames.proxy }}
+        given_name: Technical
+      - contact_type: support
+        email_address: support@{{ hostnames.proxy }}
+        given_name: Support
```
Comment (baszoetekouw) [inline roles/satosa/templates/saml2_backend.yaml.j2]: Contacts should also be configurable.  Maybe just specify them in the group_vars like
```yaml
metadata_contacts:
  - { type: "technical", name: "Pietje Puk", email: "pietje@example.org" }
  - { type: "support", name: "Jantje Jansen", email: "jantje@example.org" }
```

File: roles/satosa/defaults/main.yml [hunk 4]
```diff
@@ -13,4 +13,13 @@ satosa_port: "8080"
 satosa_state_encryption_key: "poikjh567"
 satosa_user_id_hash_salt: "59o3gaxfwg0svem1n1g3tk22i9mihcom"
 
+satosa_mdui_display_name: SATOSA Display Name
+satosa_mdui_description: SATOSA Description
+satosa_mdui_information_url: "http://information.url/"
+satosa_mdui_privacy_statement_url: "http://privacy.url/"
+satosa_mdui_keywords: "[{lang: nl, text: [sleutelwoord1, sleutelwoord2]}, {lang: en, text: [keyword1, keyword2]}]"
```
Comment (baszoetekouw) [inline roles/satosa/defaults/main.yml]: regel is te lang...

File: roles/satosa/templates/saml2_frontend.yaml.j2 [hunk 5]
```diff
@@ -1,20 +1,38 @@
...
+      display_name: {{ environment_string }} Proxy IdP
+      name: {{ environment_string }} Proxy IdP Org.
+      url: 'https://{{ hostnames.meta }}/'
+    contact_person:
+      - contact_type: technical
+        email_address: technical@{{ hostnames.proxy }}
+        given_name: Technical
+      - contact_type: support
+        email_address: support@{{ hostnames.proxy }}
+        given_name: Support
```
Comment (baszoetekouw) [inline roles/satosa/templates/saml2_frontend.yaml.j2]: zelfde opmerkingen als hierboven

File: roles/satosa/defaults/main.yml (hunk 1 above)
Comment (baszoetekouw) [inline roles/satosa/defaults/main.yml]: Misschien zijn namen als `metadata_satosa_displayname` etc overzichtelijker.

Comment (mrvanes) [reply roles/satosa/defaults/main.yml]: I don't. It should be totally clear that you're connecting to the SP or IdP from url, organisation display_name, name, SSO descriptor and endpoints... It won't help people who don't understand metadata anyway.

File: roles/satosa/templates/saml2_backend.yaml.j2 (hunk 2 above)
Comment (mrvanes) [reply roles/satosa/templates/saml2_backend.yaml.j2]: Agree

**Pull request merged**